src/testdata/json_tests/interface_mtu.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Set Interface mtu",
+  "operations":[
+    {
+      "operation": "replace",
+      "returnCode": 0,
+      "xpath": "openconfig-interfaces:interfaces/interface[name=Ethernet0]",
+      "target": "OC_YANG",
+      "attributeData": "{\"config\": {\"mtu\":9003}}"
+
+    },
+    {
+      "operation": "get",
+      "returnCode": 0,
+      "xpath": "openconfig-interfaces:interfaces/interface[name=Ethernet0]",
+      "target": "OC_YANG",
+      "type": "object",
+      "required": [
+        "openconfig-interfaces:interface"
+      ],
+      "properties": {
+        "openconfig-interfaces:interface": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": [
+              "config"
+            ],
+            "properties": {
+              "config": {
+                "type": "object",
+                "required": [
+                  "mtu"
+                ],
+                "properties": {
+                  "mtu": {
+                    "type": "integer",
+                    "enum": [
+                      9003
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  ]
+}
```

Comment:
Same thing here, should reset MTU back to its default value when done

Reply:
Added cleanup operation.